### DISCUSSION TITLE
Typo on `<Select>` documentation ?

### DISCUSSION THREAD

**Discussion by juliofarah** [radix-ui/website#635] [Q&A] [answered]
Hi! 

I just wanted to make sure this is not on oversight on my end before opening an issue or pull request, but there seems to be a small typo on the `<Select>` component documentation. https://www.radix-ui.com/docs/primitives/components/select contains `<SelectItem>` in many places, should these actually be `<Select.SelectItem>` instead?

Happy to open a PR to fix it if I'm correct.
Thanks in advance

**Comment by joaom00** [comment, marked as answer]
Hey @juliofarah,

it's not a typo, it's a component that is a little further down in the example code!

**Reply by juliofarah** [reply]
Oh I'm sorry
I've literally just scrolled down to check the rest of the code and saw that component 🤦 

Thanks for the answer anyways!
